settings_base.py

```diff
@@ -86,29 +103,62 @@
     'openoni.core',
 )
 
+# Determines how django does its caching
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
+        'LOCATION': '/var/tmp/django_cache',
+        'TIMEOUT': 4838400,  # 2 months
```

Comment:
Would you mind using this instead of the final number?
60 * 60 * 24 * 7 * 8    # 8 weeks
I like how easy it is to change that type of thing versus wondering where they got a number for 56 days.  :)